utils/pciutils/patches/110-gcc14.patch

```diff
@@ -0,0 +1,10 @@
+--- a/lib/sysfs.c
```

Comment:
I fail to find this change in the pciutils repository. Did you even try to submit all your recent GCC14 patches to upstream? Why we should carry these patches for ages?

All patches according to OpenWrt policy should be in shape to be upstreamable. Here is missing Git header with all details like who is the author, commit subject, description, etc.

Reply:
Some were upstreamed, yes.